### DISCUSSION TITLE
Minha primeira experiência com o Svelte (SvelteKit)

### DISCUSSION THREAD

**Discussion by belmirofss** [frontendbr/forum#2254] [Mostre seu trabalho]
Desde 2020 venho percebendo uma ascensão muito grande no uso do Svelte por conta de artigos e posts que vi navegando pela internet.

Até então, não havia dado muita bola a ele, porém, em um final de semana tedioso, pensei “bom… não tem nada para fazer, vamos lá!”.

Agora estou aqui para contar como foi minha primeira experiência e o que eu achei.

https://belmirofss.medium.com/minha-primeira-experi%C3%AAncia-com-o-svelte-sveltekit-daa4696f7bdf

**Comment by felquis** [comment]
Deixo aqui algumas observações para se pensar:

- 48.9kb de JavaScript.
- 7kb CSS tailwindcss.
- Está faltando um favicon bonito 😉 

### Sem JavaScript
- Ele carrega, o que é ótimo!
- List of main criptocurrencies: não funciona sem JavaScript (o que é aceitável rsrs)
  - os selects não funcionam sem JavaScript, o que é algo complicado e uma razão que eu sempre que possível escolho usar o select nativo do browser (não estou julgando seu projeto, estou julgando o atual estado de um select na web) 
- embora /criptocurrency/518 carrega sem JavaScript, parece que links de navegação não funciona, recomendo sempre usar `<a>` para navegação, e button só se for dentro de um formulário pois ele tem a função de submit e reset do form.

A graça de manter o site funcionando sem JavaScript, é para aprender mais sobre Web Platform standards e aprender a se questionar qual elemento usar, em qual momento, tipo este formulário da página inicial que é um filtro, poderia usar query string, e submeter o formulário - pois tem vários aprendizados interessantes nisso que enfim, nem sempre vão de encontro com o que o mercado em geral pratica, mas que pode acabar melhorando a forma com que a gente aborda determinadas interações em navegação, formulários, query strings em geral, e no fim das contas pode fazer vc aprender ainda mais sobre Svelte rsrs enfim, parei por aqui.

### SEO

Eu sei que o projeto é para aprender mais sobre svelte e SSR dele pelo SveltKit, mas, na hora de definir meta tags padrões ou OG e Twitter, pode ser que apareça algum aprendizado relevante também!

É isso, adorei seu projeto só dei as ideias para conversar mesmo, abs.
